chsdi/templates/htmlpopup/skitouren_metadata_test.mako

```diff
@@ -0,0 +1,19 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+   lang = lang if lang in ('fr','it','en') else 'de'
+%>
+
+<%
+   fid = str(c['featureId'])
+   image = "https://dav0.bgdi.admin.ch/swisstopoproducts/110/" + fid + ".jpg"
```

Comment:
We have a variable for that in the settings, namely `webdav_host`. Use `request.registry.settings` to access it.